docs/MD.md

```diff
@@ -1,29 +1,129 @@
 # Maskwacîs Dictionary
 
-## General Notes
-
-The TSV version of the Maskwacîs dictionary in the ALTLab repo also has semantic classifications, using the Rapid Words ontology.
+Notes on the Maskwacîs dictionary database.
 
 ## SRO
 
-* Inconsistent marking of vowel length.
+A transcription of the headword in Standard Roman Orthography (SRO).
+
+* Occasionally includes punctuation, **ex:** _ah?_ 'What? What did you say?'.
+* Vowel length is indicated with double vowels rather than macrons or circumflexes, **ex:** `aa`.
+* Vowel length is not consistently indicated. Many long vowels are written as short vowels.
+* `<i>` is (sometimes?/always?) used for unstressed short vowels (or maybe just unstressed short /a/?).
+  - **ex:** MD _wapimew_ = SRO _wâpamêw_
+* /h/ is sometimes omitted before stops (/c, p, t, k/).
+* /ou/ is written `<iw>` or `<ow>`.
+* SRO `<c>` is sometimes written as `<ch>` or `<ts>`.
+* SRO `<c>` and `<ci>` can both be written as `<ts>`.
+  - **ex:** MD _mitsow_ = SRO _mîcisow_
+* Very occasionally includes a hyphen.
+* Some entries are multiword headwords, **ex:** _ayamihew masinahikan_.
+* Imperatives are not always given in their lemma form. This might need to be reconstructed using the FST.
```

Comment:
What you mean is that there isn't always also a lemma entry for imperative verb entries.

In some case we may need to reconstruct the lemma ourselves.